package.json

```diff
@@ -10,8 +10,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@contrast/route-metrics": "github:Contrast-Security-OSS/node-route-metrics#NODE-2534-remove-preinstall-hook",
+    "body-parser": "^1.20.0",
     "express": "^4.18.1",
-    "minimist": "^1.2.6"
+    "form-data": "^4.0.0",
```

Comment:
maybe this should only be a devDependency? it looks like it's only used for tests.